chapters/develop-organize/reading/reading/README.md

```diff
@@ -18,8 +18,31 @@ To enforce the guidelines we use [automated linter checks](https://github.com/su
 Besides the mentioned guidelines, the linter implements checks for other minor style conventions (such as how to define lists, where should new lines be placed and others).
 However, for the majority of cases, the above should be sufficient.
 
+The reading content type is stored as a folder named `reading` in the course repository.
+Each topic should contain a `reading` folder that includes one or multiple Markdown files, depending on how the topic may be broken down in independent reading parts.
+The below figure highlights the placing of the reading section in a sample repository:
+
+```bash
+|-- chapter1
+|   |-- topic1
+        |-- reading
+            |-- file1.md
+            |-- file2.md
+        |-- media
+        |-- drills
+        |-- guides
+    |-- topic2
+        |-- reading
+        |-- media
+        |-- drills
+        |-- guides
+|-- chapter2
+|-- chapter3
```

Comment:
```suggestion
|-- chapter1
|   |-- topic1
        |-- reading
            |-- file1.md
            `-- file2.md
        |-- media
        |-- drills
        `-- guides
    |-- topic2
        |-- reading
        |-- media
        |-- drills
        `-- guides
|-- chapter2
`-- chapter3
```
@razvand used this pattern in the [PR for projects](https://github.com/open-education-hub/methodology/pull/32/files#diff-4c1be4df898340835324c5bcda5429cb7c72517b46ab4e42312de68275600a90R56).